node/resolvers/directives/withPermissions.ts

```diff
@@ -4,6 +4,15 @@ import type { GraphQLField } from 'graphql'
 import { defaultFieldResolver } from 'graphql'
 import { SchemaDirectiveVisitor } from 'graphql-tools'
 
+import type StorefrontPermissions from '../../clients/storefrontPermissions'
+
+export const getCheckUserPermission = async (
```

Comment:
We are just getting the user permissions here right? Should we call it just  `getUserPermission`?

Reply:
I agree, I let the same previous name be created. But I will change.